ZUGFeRD/AdditionalReferencedDocument.cs

```diff
@@ -55,5 +55,10 @@ public class AdditionalReferencedDocument : BaseReferencedDocument
         /// Type of the reference document
         /// </summary>
         public AdditionalReferencedDocumentTypeCode TypeCode { get; set; }
+
+        /// <summary>
+        /// MIME code of the attached document
+        /// </summary>
+        public string MimeCode { get; set; }

```

Comment:
Themime code should be extracted automatically using MimeTypeMapper, can you please modify this?